### DISCUSSION TITLE
Feature request: Viewing file's Alternate Data Streams in Explorer

### DISCUSSION THREAD

**Discussion by getCursorsExe** [valinet/ExplorerPatcher#2483] [Ideas]
You don't have to use command line to show Alternate Data Streams.

**Comment by pyrates999** [comment]
I'm gonna say this is a no.

**Reply by getCursorsExe** [reply]
This relates to security exploit prevention, such as using Alternate Data Streams to spoof a 5 GB file as 1 MB file, which is a very bad way to use Alternate Data Streams.
